pkg/sdk/testint/schemas_integration_test.go

```diff
@@ -81,16 +81,16 @@ func TestInt_SchemasCreate(t *testing.T) {
 	})
 
 	t.Run("with tags", func(t *testing.T) {
-		tagName := random.String()
-		tagID := sdk.NewAccountObjectIdentifier(tagName)
+		tagID := testClientHelper().Ids.RandomAccountObjectIdentifier()
+		tagName := tagID.Name()
 		_, err := client.ExecForTests(ctx, fmt.Sprintf(`CREATE TAG "%s"`, tagName))
```

Comment:
It's interesting that it works, because as far as I remember tags are objects create on schema. I guess, some database + schema are taken from session context. Maybe we should mark it as something to do in tags refactoring ?

Reply:
It works, because our test clients have db and schema context (and we are reusing db and schema in the majority of tests). I will use the create tag from helper, because that one uses the fully qualified name, so we will be safer in case we change the client context approach (in one of the next PRs, we still have to target all ExecForTests invocations and replace the with helper methods, but it was supposed to be don iteratively)